### PR TITLE
Update to check running pods are on same node

### PR DESCRIPTION
**Pull request by jilju**
tests/manage/pv_services/test_verify_rwo_using_dc_pod.py
1. Update to check whether running pods are on same node.
2. Increase replica count to verify  using more pods.

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

File: tests/manage/pv_services/test_verify_rwo_using_dc_pod.py [hunk 1]
```diff
@@ -75,40 +73,50 @@ def wait_for_pods_and_verify(self):
         """
         # Wait for pods
         for pods in TimeoutSampler(
-            180, 2, func=pod.get_all_pods, namespace=self.namespace,
+            360, 2, func=pod.get_all_pods, namespace=self.namespace,
             selector=[self.name], selector_label='name'
         ):
-            if len(pods) == 2:
+            if len(pods) == 5:
```
Comment (AviadP) [inline tests/manage/pv_services/test_verify_rwo_using_dc_pod.py:79]: if meaning of this condition is `if len(pods) == replica_count` that defined in another function, plz consider using constant instead

File: tests/manage/pv_services/test_verify_rwo_using_dc_pod.py [hunk 2]
```diff
@@ -75,40 +73,50 @@ def wait_for_pods_and_verify(self):
...
+                    f"Pods which are running are on the same node "
+                    f"{pod_running_node}"
+                )
+            except ResourceWrongStatusException:
+                log.info(
+                    f"Verified: Pod {pod_obj.name} is not in "
+                    f"running state."
+                )
 
     def test_verify_rwo_using_replica_2_dc_pod(self):
```
Comment (sidhant-agrawal) [inline tests/manage/pv_services/test_verify_rwo_using_dc_pod.py:121]: shouldn't it be `test_verify_rwo_using_replica_5_dc_pod`?

File: tests/manage/pv_services/test_verify_rwo_using_dc_pod.py (hunk 1 above)
Comment (jilju) [reply tests/manage/pv_services/test_verify_rwo_using_dc_pod.py]: Done

File: tests/manage/pv_services/test_verify_rwo_using_dc_pod.py (hunk 2 above)
Comment (jilju) [reply tests/manage/pv_services/test_verify_rwo_using_dc_pod.py]: Modified the test name

**Review by ebenahar — APPROVED** [review]

**Pull request merged**